### PR TITLE
Added clarification for ARM Dashboard deployments

### PR DESCRIPTION
**Pull request by sesispla**
Current documentation was not clear for non amd64 platforms. Added an small clarification and command example to deploy it to ARM systems.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2105)

### CODE REVIEW

File: docs/user-guide/ui.md [hunk 1]
```diff
@@ -17,12 +17,17 @@ Dashboard also provides information on the state of Kubernetes resources in your
...
+The Dashboard UI is not deployed by default. To deploy it on amd64 servers, run the following command:
 
 ```
 kubectl create -f https://rawgit.com/kubernetes/dashboard/master/src/deploy/kubernetes-dashboard.yaml
 ```
 
+When deploying to platforms other than amd64 consider execute this command instead. This example replaces the default amd64 docker image for the arm one on the fly:
+
+```
+curl -sSL 'https://rawgit.com/kubernetes/dashboard/master/src/deploy/kubernetes-dashboard.yaml?raw=true' | sed 's/amd64/arm/g' | kubectl create -f -
```
Comment (rf232) [inline docs/user-guide/ui.md]: I don't think the ?raw=true query parameter is necessary for a request to rawgit.com

File: docs/user-guide/ui.md [hunk 2]
```diff
@@ -17,12 +17,17 @@ Dashboard also provides information on the state of Kubernetes resources in your
 
 ## Deploying the Dashboard UI
 
-The Dashboard UI is not deployed by default. To deploy it, run the following command:
+The Dashboard UI is not deployed by default. To deploy it on amd64 servers, run the following command:
```
Comment (devin-donnelly) [inline docs/user-guide/ui.md]: "To deploy it on amd64-based servers, run the following command:"

File: docs/user-guide/ui.md [hunk 3]
```diff
@@ -17,12 +17,17 @@ Dashboard also provides information on the state of Kubernetes resources in your
...
 
 ```
 kubectl create -f https://rawgit.com/kubernetes/dashboard/master/src/deploy/kubernetes-dashboard.yaml
 ```
 
+When deploying to platforms other than amd64 consider execute this command instead. This example replaces the default amd64 docker image for the arm one on the fly:
```
Comment (devin-donnelly) [inline docs/user-guide/ui.md]: "If you want to deploy Dashboard on a platform other than amd64 (such as ARM), use the following command:"

Add after the example command: "The above command replaces the default amd64 Docker image with an ARM Docker image."

**Comment by sesispla** [comment]
@rf232 all suggestions applied and merge from master.

**Comment by sesispla** [comment]
Added some changes reflecting the changes introduced in this Dashboard PR: [#1549](https://github.com/kubernetes/dashboard/pull/1549)

**Comment by chenopis** [comment]
@rf232 Any further comments or suggestions?

**Comment by chenopis** [comment]
@rf232 @devin-donnelly Is this good to go?

**Comment by jaredbhatti** [comment]
This appears valid, although it's weird that we lead with the exceptions first (amd and arm) and then end with the default (which I guess is Intel?).  From a docs perspective, the PR is accurate, but that section could probably use a rewrite to lead with the default first and then the exceptions.

**Comment by sesispla** [comment]
Just updated my fork and checked that all changes where already applied to the master branch. I think we can close this PR.

Thx

**Pull request closed without merging**